### ISSUE TITLE
Support for tsaklist

### ISSUE THREAD

**Issue by liguangsheng** [trentm/python-markdown2#216]
How can I convert 

```
- [x] item1
- [ ] item2
```
to
```
<ul class="contains-task-list">
<li class="task-list-item">
<input type="checkbox" class="task-list-item-checkbox" checked disabled> item1
</li>
<li class="task-list-item">
<input type="checkbox" class="task-list-item-checkbox" disabled> item2
</li>
</ul>
```
which looks like
![default](https://cloud.githubusercontent.com/assets/8074650/16298815/ca17ab7e-3969-11e6-8894-7d294e3d213e.PNG)

**Comment by nicholasserra** [comment]
That's a github markdown feature that we do not have implemented. PRs welcome :)

**Comment by nicholasserra** [comment]
Fixed in #218
